tests/commonTemplates_test.go

```diff
@@ -208,6 +208,30 @@ var _ = Describe("Common templates", func() {
 	})
 
 	Context("resource change", func() {
+
+		Context("namespace change", func() {
+			AfterEach(func() {
+				strategy.RevertToOriginalSspCr()
+			})
+
+			It("[test_id:6057] should not fail to update commonTemplates.namespace", func() {
+				foundSsp := getSsp()
+				const testNamespace = "ssp-operator-functests"
+				Expect(foundSsp.Spec.CommonTemplates.Namespace).ToNot(Equal(testNamespace), "namespaces should not equal")
+				foundSsp.Spec.CommonTemplates.Namespace = testNamespace
+
+				err := apiClient.Update(ctx, foundSsp)
```

Comment:
This update may fail. It is more stable to call `updateSsp()` that retries updates:
```go
updateSsp(func(foundSsp *sspv1beta1.SSP) {
	foundSsp.Spec.CommonTemplates.Namespace = testNamespace
})
```